### PR TITLE
New advenue header bidding adapter

### PR DESCRIPTION
**Pull request by devadvenue**
## Type of change

- [x] New bidder adapter   

## Description of change

Second pull request to merge without package-lock.json
https://github.com/prebid/Prebid.js/pull/3321

- test parameters for validating bids
```
{
   	bidder: 'advenue',
   	params: {
      		placementId: 0,
      		traffic: 'banner'
    	}
}
```
- contact email of the adapter’s maintainer ssp@advenuemedia.co.uk
- [x] official adapter submission

## Other information

Docs prebid.github.io : https://github.com/prebid/prebid.github.io/pull/1046

### CODE REVIEW

**Comment by devadvenue** [comment]
After creating new pull request merge conflict with package-lock steal remains, so original file was uploaded to resolve merge conflict

File: modules/advenueBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    let winTop;
+    try {
+      winTop = window.top;
+      winTop.location.toString();
+    } catch (e) {
+      utils.logMessage(e);
+      winTop = window;
+    };
+
+    const location = winTop.location;
```
Comment (jsnellbaker) [inline modules/advenueBidAdapter.js]: You could likely use the `bidderRequest.refererInfo` property to help obtain some of this information.

You can find additional details here: http://prebid.org/dev-docs/bidder-adaptor.html#referrers

**Comment by devadvenue** [comment]
update

**Comment by jsnellbaker** [comment]
@devadvenue Thanks for making the updates; LGTM now.

**Pull request merged**